jsonnet/telemeter/rules.libsonnet

```diff
@@ -35,6 +35,18 @@
                 topk by (_id) (1, max by (_id, managed, ebs_account, internal) (label_replace(label_replace((subscription_labels{support=~"Standard|Premium|Layered"} * 0 + 1) or subscription_labels * 0, "internal", "true", "email_domain", "redhat.com|(.*\\.|^)ibm.com"), "managed", "", "managed", "false")) + on(_id) group_left(version) (topk by (_id) (1, 0*cluster_version{type="current"})))
               |||,
             },
+            {
+              record: 'id:alerts:count',
```

Comment:
It may be misleading to call this recorded metric `id:alerts:count` as it records specifically only the alerts that are in `firing` state and with a severity of `critical`

Reply:
Would {alertstate='firing', severity='critical'} labels be copied to the new metric?
If yes it _might_ be ok, similar to how only `alerts{alertstate='firing'}` subset is sent to telemetry but the metric is still  called just `alerts`.

(I don't have a qualified opinion, don't know much about metric naming conventions)

Reply:
They won't. But you can add them to the labels of the recording rules just as well:

```
{
              record: 'id:alerts:count',
              expr: |||
                count by (_id) (alerts{alertstate='firing', severity='critical'})
              |||,
              labels: {
                alertstate: "firing",
                severity: "critical",
              }
            },